VideoApp/VideoApp/Video/Room/ConnectOptionsFactory.swift

```diff
@@ -75,6 +63,16 @@ import TwilioVideo
     }
 }
 
+private extension TwilioVideo.VideoCodec {
+    static func make(setting: VideoCodec) -> TwilioVideo.VideoCodec {
```

Comment:
I couldn't use `init` for this like others I think because VideoCodec is a base class. This is fine for now but there may be a nicer way to handle this stuff currently done in extensions.